data/sql/updates/pending_db_world/rev_1616252599323979000.sql

```diff
@@ -0,0 +1,80 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1616252599323979000');
+/*
+ * Zone: Terokkar Forest
+ * Update by Knindza | <www.azerothcore.org>
+*/
+
+/* SMARTSCRIPT */
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 21787;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 21787);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21787, 0, 0, 0, 11, 0, 100, 1, 0, 0, 0, 0, 11, 16592, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Time-Lost Skettis High Priest - On Respawn - Cast \'16592\' (No Repeat)'),
+(21787, 0, 1, 0, 4, 0, 100, 1, 0, 0, 0, 0, 11, 11639, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Time-Lost Skettis High Priest - On Aggro - Cast \'11639\' (No Repeat)'),
+(21787, 0, 2, 0, 0, 0, 100, 0, 2300, 3000, 7800, 9000, 11, 9734, 0, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Time-Lost Skettis High Priest - In Combat - Cast \'9734\''),
+(21787, 0, 3, 0, 2, 0, 100, 1, 10, 30, 0, 0, 11, 42420, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Time-Lost Skettis High Priest - Between 10-30% Health - Cast \'42420\' (No Repeat)');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 21644;
+DELETE FROM `smart_scripts` WHERE (source_type = 0 AND entryorguid = 21644);
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(21644, 0, 0, 0, 0, 0, 100, 0, 1000, 3000, 30000, 31000, 11, 13730, 32, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Skettis Wing Guard - In Combat - Cast \'13730\''),
+(21644, 0, 1, 0, 0, 0, 100, 0, 2500, 5000, 20000, 25000, 11, 35321, 33, 0, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 'Skettis Wing Guard - In Combat - Cast \'35321\''),
+(21644, 0, 2, 0, 2, 0, 100, 1, 20, 80, 0, 0, 11, 40546, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Skettis Wing Guard - Between 20-80% Health - Cast \'40546\' (No Repeat)'),
+(21644, 0, 3, 4, 2, 0, 100, 1, 5, 30, 0, 0, 11, 8599, 1, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Skettis Wing Guard - Between 5-30% Health - Cast \'8599\' (No Repeat)'),
+(21644, 0, 4, 0, 61, 0, 100, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 'Skettis Wing Guard - Between 5-30% Health - Say Line 0 (No Repeat)');
```

Comment:
It already has AI

Reply:
I don't know if it's better than it is now

Reply:
Yeah, those i updated had kinda bad AI.
Overlaping with spells mostly.